rmc/utils/constraint.py

```diff
@@ -242,7 +240,7 @@ def calculate_exp_from_mu(
             context_ht.modifier,
             context_ht.transcript,
             context_ht.coverage,
-        ].expected_per_variant,
+        ].expected,
```

Comment:
adding this here again in case the other comment gets lost -- we could maybe switch this to be an `ht.index` https://github.com/broadinstitute/gnomad_methods/blob/main/gnomad/utils/constraint.py#L46